tst/EnergyPlus/unit/ConvectionCoefficients.unit.cc

```diff
@@ -1242,3 +1242,139 @@ TEST_F(ConvectionCoefficientsFixture, CalcBeausoleilMorrisonMixedUnstableCeiling
+
+    ConvectionCoefficient = CalcASHRAESimpleIntConvCoeff(Tsurf, Tamb, CosTilt);
+    EXPECT_EQ(ConvectionCoefficient, ExpectedCoefficient);
+
+    // Scenario: Horizontal Surface with reduced convection
+    // Hcov expected = 0.948
+    // A negative Delta_T is required for reduced convection
+
+    Tsurf = 30.0;
+    Tamb = 20.0;
+    CosTilt = 0.9239; // cos(22.5 degrees)
+    ExpectedCoefficient = 0.948;
+
+    ConvectionCoefficient = CalcASHRAESimpleIntConvCoeff(Tsurf, Tamb, CosTilt);
+    EXPECT_EQ(ConvectionCoefficient, ExpectedCoefficient);
+
+
+    //Scenario: Horizontal surface with enhanced convection:
+    // Hcov expected = 4.040
+    // A positive Delta_T is required for enhanced convection
+
+    Tsurf = 20.0;
+    Tamb = 30.0;
+    CosTilt = 0.9239; // cos(22.5 degrees)
+    ExpectedCoefficient = 4.040;
+
+    ConvectionCoefficient = CalcASHRAESimpleIntConvCoeff(Tsurf, Tamb, CosTilt);
+    EXPECT_EQ(ConvectionCoefficient, ExpectedCoefficient);
+
+    //Scenario: horizontal surface, enhanced convection
+    // 180 degree surface, negative Delta_T
+    // Hcov expected = 4.040
+
+    Tsurf = 30.0;
+    Tamb = 20.0;
+    CosTilt = -1; // cos(180 degrees)
+    ExpectedCoefficient = 4.040;
+
+    ConvectionCoefficient = CalcASHRAESimpleIntConvCoeff(Tsurf, Tamb, CosTilt);
+    EXPECT_EQ(ConvectionCoefficient, ExpectedCoefficient);
+
+    //Scenario: horizontal surface, reduced convection
+    // 180 degree surface, positive Delta_T
+    // Hcov expected = 0.948
+
+    Tsurf = 20.0;
+    Tamb = 30.0;
+    CosTilt = -1; // cos(180 degrees)
+    ExpectedCoefficient = 0.948;
+
+    ConvectionCoefficient = CalcASHRAESimpleIntConvCoeff(Tsurf, Tamb, CosTilt);
+    EXPECT_EQ(ConvectionCoefficient, ExpectedCoefficient);
+
+    //Scenario: tilted surface with reduced convection
+    // Hcov expected = 2.281
+    // A negative Delta_T is required for reduced convection
+
+    Tsurf = 30.0;
+    Tamb = 20.0;
+    CosTilt = 0.707; // cos(45 degrees)
+    ExpectedCoefficient = 2.281;
+
+    ConvectionCoefficient = CalcASHRAESimpleIntConvCoeff(Tsurf, Tamb, CosTilt);
+    EXPECT_EQ(ConvectionCoefficient, ExpectedCoefficient);
+
+    //Scenario: tilted surface with enhanced convection
+    // Hcov expected = 3.870
+
+    Tsurf = 20.0;
+    Tamb = 30.0;
+    CosTilt = 0.707; // cos(45 degrees)
+    ExpectedCoefficient = 3.870;
+
+    ConvectionCoefficient = CalcASHRAESimpleIntConvCoeff(Tsurf, Tamb, CosTilt);
+    EXPECT_EQ(ConvectionCoefficient, ExpectedCoefficient);
+}
+
+TEST_F(ConvectionCoefficientsFixture, ConvectionCoefficientsTest_HConvInDependence)
+{
+    Real64 ConvectionCoefficient;
+    Real64 ExpectedCoefficient = 3.076;
+
+    DataSurfaces::Surface.allocate(1);
+    DataSurfaces::Surface(1).CosTilt = 0;
+
+    DataHeatBalance::HConvIn.allocate(1);
+
+    CalcASHRAESimpleIntConvCoeff(1, 20.0, 30.0);
+
+    ConvectionCoefficient = DataHeatBalance::HConvIn(1);
+
+    EXPECT_EQ(ConvectionCoefficient, ExpectedCoefficient);
+}
```

Comment:
Great set of unit tests.  Kind of a shame that you missed one code path in all the possible configurations (line 3874 [here](http://energyplus.s3-website-us-east-1.amazonaws.com/coverage/2020-05-15-EnergyPlus-9bf737ca59-PR7890-x86_64-gcc-7.5UTCD/EnergyPlus/ConvectionCoefficients.cc.gcov.html)), but again, not going to hold this up for that, this is good work.